.goreleaser.Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM alpine
```

Comment:
Do we want any labels on our docker image? Maybe one for Maintainer? Or maybe information about the actual image: source / revision / build date/ ... ? 

I've used some [open containers](https://github.com/opencontainers/image-spec/blob/main/annotations.md)  standards in other repos, but not sure they are needed for now. We could always follow up with that later - or we can add them like in https://github.com/frankywahl/super_hooks/blob/master/.goreleaser.Dockerfile#L3-L11